### PR TITLE
add kube-state-metrics  to cluster/cluster-metrics

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
add `kube-state-metrics` to cluster-metrics

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```NONE
```

### CODE REVIEW

**Comment by zouyee** [comment]
ping @eparis

File: cluster/addons/cluster-metrics/kube-state-metrics-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-state-metrics
+    spec:
+      serviceAccountName: kube-state-metrics
+      containers:
+      - name: kube-state-metrics
+        image: quay.io/coreos/kube-state-metrics:v1.0.1
```
Comment (eparis) [inline cluster/addons/cluster-metrics/kube-state-metrics-deployment.yaml]: It doesn't appear that we ship addons which pull from anywhere other than gcr.io. Should we start?

**Review by eparis — COMMENTED** [review]
I don't know anything about kube-state-metrics. So its hard to know if this belongs.

**Pull request closed without merging**